administration/config/modules.ts

```diff
@@ -49,23 +49,20 @@ function getAdditionalModulePaths(paths: ReturnType<typeof getPaths>, options: {
 /**
  * Get webpack aliases based on the baseUrl of a compilerOptions object.
  */
-function getWebpackAliases(paths: ReturnType<typeof getPaths>, options: { baseUrl?: string } = {}) {
+const getWebpackAliases = (paths: ReturnType<typeof getPaths>, options: { baseUrl?: string } = {}) => {
   const baseUrl = options.baseUrl
 
-  if (!baseUrl) {
-    return {}
-  }
+  const baseUrlResolved = baseUrl ? path.resolve(paths.appPath, baseUrl) : null
 
-  const baseUrlResolved = path.resolve(paths.appPath, baseUrl)
-
-  if (path.relative(paths.appPath, baseUrlResolved) === '') {
+  if (baseUrlResolved && path.relative(paths.appPath, baseUrlResolved) === '') {
     return {
       src: paths.appSrc,
     }
   }
+  return {}
```

Comment:
odd as in code style wise or odd as in wrong? :D I tried to avoid another return but I guess you are right, this is not very self explanatory. Reverted it and just added another return.

Reply:
Yeah, I thought the previous version was a bit easier to read... But that's just taste I guess :D

Reply:
Upon looking at the code I wrote again I agree :D